src/app/components/PortraitVideoCarousel/index.test.tsx

```diff
@@ -12,8 +12,8 @@ describe('PortraitVideoCarousel', () => {
     });
 
     const portraitVideoItems = (container as unknown as HTMLElement)
-      .querySelector('div[data-testid="pv-scroll-panel"]')
-      ?.querySelectorAll('button');
+      .querySelector('ul[data-testid="pv-carousel"]')
```

Comment:
Could we use the react testing library `getByTestId` instead of using the native js query selector please?